src/oneD/refine.cpp

```diff
@@ -12,7 +12,7 @@ namespace Cantera
 {
 Refiner::Refiner(Domain1D& domain) :
     m_ratio(10.0), m_slope(0.8), m_curve(0.8), m_prune(-0.001),
-    m_min_range(0.01), m_domain(&domain), m_npmax(3000),
+    m_min_range(0.01), m_domain(&domain), m_npmax(300),
```

Comment:
I should note, though, that the default argument for `Sim1D::setMaxGridPoints` sets the max to 300: https://github.com/Cantera/cantera/blob/master/include/cantera/oneD/Sim1D.h#L136